ANSWERS.md

```diff
@@ -0,0 +1,11 @@
+1.  What are PropTypes used for?
+Proptypes are used for typechecking passed props to make sure that their data types are correct. It helps keep code organized and helps future developers understand the code. 
+
+2.  What is a lifecycle event in React?
+It's one part in the process in the rendering, displaying, updating, and removing of a component to and from the DOM. In React it's called mounting and unmounting. These events are open to being overridden and having code run on them.
```

Comment:
also it's called updating, and rendering. Remember that just because something receives new props, doesn't necessarily mean it's mounting or re-rendering.